WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/addons/AddonRepository.kt

```diff
@@ -0,0 +1,43 @@
+package com.woocommerce.android.ui.products.addons
+
+import com.woocommerce.android.annotations.OpenClassOnDebug
+import com.woocommerce.android.model.Order.Item.Attribute
+import com.woocommerce.android.model.toAppModel
+import com.woocommerce.android.tools.SelectedSite
+import org.wordpress.android.fluxc.model.WCOrderModel
+import org.wordpress.android.fluxc.model.order.OrderIdentifier
+import org.wordpress.android.fluxc.store.WCOrderStore
+import org.wordpress.android.fluxc.store.WCProductStore
+import javax.inject.Inject
+
+@OpenClassOnDebug
+class AddonRepository @Inject constructor(
+    private val orderStore: WCOrderStore,
+    private val productStore: WCProductStore,
+    private val selectedSite: SelectedSite
+) {
+    fun fetchOrderAddonsData(
+        orderID: Long,
+        orderItemID: Long,
+        productID: Long
+    ) = fetchOrder(orderID)
+        ?.findOrderAttributesWith(orderItemID)
+        ?.joinWithAddonsFrom(productID)
+
+    private fun fetchOrder(orderID: Long) =
```

Comment:
🔧 Can we change the method name to `get` ? If I see correctly, `WCOrderStore#getOrderByIdentifier` gets data from database and `fetch` I think is used usually when referring to fetching data from remote. 

```suggestion
    private fun getOrder(orderID: Long) =
```

p1627037388457800-slack-C02QANACA

Reply:
Agreed! Fixed here: https://github.com/woocommerce/woocommerce-android/pull/4590/commits/fa6b77588fe31af3f7407f7a6564bc98dcfbc815